generator/src/lib.rs

```diff
@@ -2334,10 +2348,16 @@ pub struct ConstantMatchInfo {
     pub is_alias: bool,
 }
 
-pub fn generate_const_debugs(
-    const_values: &BTreeMap<Ident, Vec<ConstantMatchInfo>>,
-) -> TokenStream {
+#[derive(Default)]
+pub struct ConstantTypeInfo {
+    values: Vec<ConstantMatchInfo>,
+    // TODO: Could also be `type: Option<Ident>,`
+    bitwidth: Option<u32>,
```

Comment:
I don't really remember why I wanted this, using an integer is probably more consistent. We can always change it to contain the type in the future, necessary at all.

Reply:
In hindisght, the type name is the _key_ in `const_values: &BTreeMap<Ident, ConstantTypeInfo>`.